examples/agent-sandbox-rl/agent_sandbox_rl/fleet.py

```
@@ -657,6 +657,42 @@ def warm_image(self, image: str, *, replicas_override: int | None = None,
+    workers = max(1, min(len(entries), self.config.max_concurrent))
+    if workers == 1 or len(entries) == 1:
+      for e in entries:
+        self._unwarm_entry(e)
+      return
+    with ThreadPoolExecutor(max_workers=workers) as ex:
+      futures = [ex.submit(self._unwarm_entry, e) for e in entries]
+      for f in as_completed(futures):
+        try:
+          f.result()
+        except Exception:  # noqa: BLE001
+          pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)